### DISCUSSION TITLE
sub-menu of search: the field "number of typed photos" is partially empty

### DISCUSSION THREAD

**Discussion by loulou91** [photoprism/photoprism#3593] [Q&A]
sub-menu of search allows the display of typed set of photos (monochrome, animated...). The number of each individual set looks to be empty if no photos in it. But if they are not empty only "Review" set has a figure. The others are blank even if there are some photos in it.
Is it normal?

**Comment by lastzero** [comment]
Please note that most navigation links are provided for your convenience and simply use the existing search with a preset filter. For performance reasons, there are no exact counts for each search filter type and combination.

**Comment by loulou91** [comment]
Here, I don't really care if counts are exact or not, my point was how to understand blanks. As an example for "review" blank means 0 but for monochrome blank means "no value". It should be more coherent to display "0" instead of blank when count is available.

**Reply by lastzero** [reply]
I see. That should be possible although this is the first time such a request comes up.

**Comment by loulou91** [comment]
Do as you want, it's only a small remark but take note that there is always a first time! ;-)
